integration-tests/s3_test.go

```diff
@@ -0,0 +1,239 @@
+package integration_tests_test
+
+import (
+	"fmt"
+
+	testframework "github.com/cloudfoundry/cloud-service-broker/brokerpaktestframework"
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	. "github.com/onsi/gomega/gstruct"
+)
+
+var customS3Plans = []map[string]any{
+	customS3Plan,
+}
+
+var customS3Plan = map[string]any{
+	"name":        "custom-plan",
+	"id":          "9dfa265e-1c4d-40c6-ade6-b341ffd6ccc3",
+	"description": "custom S3 plan defined by customer",
+	"acl":         "private",
+	"metadata": map[string]any{
+		"displayName": "custom S3 service",
+	},
+	"labels": map[string]any{
```

Comment:
labels are not a parameter that can be defined for a plan

Reply:
Done!! Thank you very much!!